python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -260,9 +260,11 @@ class DiskBasedFeature(Feature):
     def __init__(self, path: str, metadata: Dict = None):
         super().__init__()
         mmap_mode = "r+"
-        self._tensor = torch.from_numpy(
-            np.load(path, mmap_mode=mmap_mode)
-        ).contiguous()
+        loaded_array = np.load(path, mmap_mode=mmap_mode)
+        assert not np.isfortran(
```

Comment:
Extracted from the official doc: `This function is obsolete ...`.
Let's use flags instead: https://numpy.org/doc/stable/reference/generated/numpy.ndarray.flags.html